portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -44,17 +49,16 @@
 
   @Override
   public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
-    System.out.println(request.getParameter("lang"));
-
     List<String> comments = new ArrayList<>();
     Query query = new Query(ENTITY_KIND).addSort(ENTITY_TIMESTAMP_HEADER, SortDirection.ASCENDING);
     PreparedQuery results = datastore.prepare(query);
-
+    String prevLanguageCode = request.getParameter("sourceLanguageCode");
+    String languageCode = request.getParameter("targetLanguageCode");
```

Comment:
Try to be consistent in the file. So you should use sourceLanguageCode/targetLanguageCode as the string variable names.

Reply:
Done